x-pack/legacy/plugins/infra/public/pages/infrastructure/snapshot/toolbar.tsx

```diff
@@ -106,7 +106,7 @@ export const SnapshotToolbar = injectI18n(({ intl }) => (
                     groupBy={groupBy}
                     nodeType={nodeType}
                     onChange={changeGroupBy}
-                    fields={derivedIndexPattern.fields}
+                    fields={derivedIndexPattern.fields as any}
```

Comment:
We should probably fix our types so that `searchable` and `aggregatable` are optional fields to match the `FieldType` properties, to avoid this `any`. I'm looking into whether we can do that easily...

Reply:
OK @ppisljar for whatever reason I can't push to this branch so I opened a PR against yours: